### DISCUSSION TITLE
Sort Sections in 'Language Server Configurations'

### DISCUSSION THREAD

**Discussion by silvasch** [helix-editor/helix#11297] [General]
### Summary

In the section `Language Server Configurations` of the wiki, the sidebar seems to be ordered alphabetically. However, some entries are in the wrong place:

- `PHP` sits between `HTML` and `JavaScript`
- `Scala` and `SQL` are between `Markdown` and `Markdoc`
- `OpenPolicyAgent` is after all the `Python`'s
- `WGSL` is before `V`

### Reproduction Steps

-

### Helix log

-

### Platform

-

### Terminal Emulator

-

### Installation Method

-

### Helix Version

-

**Comment by pascalkuthe** [comment]
The wiki is not part of the repository. It's free to edit for anyone.
